config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -1367,3 +1367,36 @@ periodics:
     testgrid-dashboards: sig-node-kubelet
     testgrid-tab-name: gcp-kubelet-credential-provider
     description: "tests feature gcp kubelet image credential provider"
+- name: ci-cos-inplace-pod-resize-containerd-main-e2e-gce
+  interval: 24h
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-e2e-containerd: "true"
+  spec:
+    containers:
+    - args:
+      - --repo=github.com/containerd/containerd=main
```

Comment:
```suggestion
      - --repo=k8s.io/kubernetes=master
      - --repo=github.com/containerd/containerd=main
```

Reply:
I don't think this is needed, see the other cluster e2e job as example (https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-node/containerd.yaml#L1018-L1048)

I think specifying `k8s.io/kubernetes=master` is only needed for node e2e jobs, we don't need it for cluster e2e jobs.

Reply:
👍